Addresses issues 147-166

KrishnaTO opened the pull request:
See individual commits

closes #147
closes #148
closes #149
closes #152
closes #153
closes #155
closes #156
closes #158
closes #160
closes #161
closes #162
closes #164
closes #165
closes #166

KrishnaTO committed:
#147 added class 'live fence'

KrishnaTO committed:
added relation to hedge

KrishnaTO committed:
#148 added class 'boundary planting'

KrishnaTO committed:
Merge branch 'issue-147' into issue-148

KrishnaTO committed:
Added synonym 'fenceline planting'

KrishnaTO committed:
updated definition

"...or hedges..."

KrishnaTO committed:
#149 Added classes per 'windbreak'

Added classes:
- Windbreak
- timberbelt
- buffer planting

KrishnaTO committed:
#152  added class 'hedgerow trees'

KrishnaTO committed:
#153 added class 'fodder bank'

KrishnaTO committed:
#155 added class 'farm woodland'

KrishnaTO committed:
#156 added class 'scattered trees'

KrishnaTO committed:
#158  added class 'taungya farming'

KrishnaTO committed:
#160 added class 'forest garden'

KrishnaTO committed:
#161 added class 'shifting cultivation'

KrishnaTO committed:
#162 added class 'forest grazing'

KrishnaTO committed:
#164 added relation to 'hedges'

KrishnaTO committed:
#165 added comments to 'alley cropping'

KrishnaTO committed:
#166 added relation to 'shifting cultivation'

KrishnaTO committed:
Merge branch 'issue-148' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-149' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-152' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-153' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-155' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-156' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-158' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-160' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-161' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-162' into issues-147-166

KrishnaTO committed:
Merge branch 'issue-165' into issues-147-166

celineaubert commented on src/ontology/agro-edit.owl:
```diff
@@ -4679,6 +4692,8 @@ AnnotationAssertion(Annotation(<http://purl.org/dc/elements/1.1/source> "https:/
 AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00000579> <https://orcid.org/0000-0003-3985-4911>)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/AGRO_00000579> "alleycropping"@en)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/AGRO_00000579> "hedgerow intercropping"@en)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/AGRO_00000579> "A cropping system that involves growing crops in a wide strip, typically 6 meters in width, between lines of closely planted, fast-growing trees or shrubs."@en)
```
where does this comment come from?

KrishnaTO replied:
From #165 , where a revised definition was proposed, it was added as a comment due to the indefinite but typical content.

celineaubert committed:
reviewed #147

changed creator ID
added a subclass of

celineaubert commented on src/ontology/agro-edit.owl:
```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010157> "https://orcid.org/0000-0003-3985-4911")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010157> "2021-12-27T16:54:13Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010157> "boundary planting"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010157> <http://purl.obolibrary.org/obo/AGRO_00000231>)
```
I wonder if this term should go under ''live fence'' class. I have found that exact syn are:  barrier planting, Trees on boundaries and live fence. 
The term is a bit tricky as it includs a verb. But I think the term does not describe a process but the finality of this process (based on the other term in ERA list), and is therefore a continuant.  

Definition: Live fence made of trees. Boundary planting are used to delimitate property, agricultural fields, pastures, roads or any other place where fences might be located.

Source: Baker, K., Baumann, D., Michaud, J.S., van Mossel-Forrester, B., 2014. Technical Specification Boundary Planting. Registered Plan Vivo Project: The CommuniTree Carbon Project. 64p. URL: https://takingroot.org/wp-content/uploads/2015/04/takingroot-boundary-planting.pdf

celineaubert commented on src/ontology/agro-edit.owl:
```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010158> "2022-01-03T23:59:15Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/AGRO_00010158> "shelterbelt")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010158> "windbreak"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010158> <http://purl.obolibrary.org/obo/AGRO_00000584>)
```
I wonder if the subcass: http://purl.obolibrary.org/obo/ENVO_01000408 would be better? Again, this is not a process but the result of the process of planting trees.

celineaubert committed:
added hedgerow intercropping

not linked to any opened issue. I just saw the term when looking for another one and thought f adding it to AgrO.

celineaubert commented on src/ontology/agro-edit.owl:
```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/w/index.php?title=Hedge&oldid=1062906324") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/AGRO_00010161> "Tree plantings which are grown in hedgerows but have been allowed to reach their full height and width."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/AGRO_00010161> <https://orcid.org/0000-0003-3985-4911>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/AGRO_00010161> "2022-01-04T00:23:05Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/AGRO_00010161> "production hedges")
```
I have asked Peter the definition of this term that does not appear in the source he mentionned neither on any other document.

celineaubert committed:
move scattered tree under Slivopasture

as the def for this terms include animal shelter and grazing. In agrisilviculture, no animal are included.

celineaubert committed:
reviewed #156

reviewed the def of the term based on the source provided by Peter.
Also added a new term based on the orginal term mentionned by Peter

celineaubert committed:
rewied #160

reviewed the definition and label

celineaubert commented on src/ontology/agro-edit.owl:
```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010166> "forest garden"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010166> <http://purl.obolibrary.org/obo/AGRO_00000584>)
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010167> (shifting cultivation)
```
Could you please import the PATO term : http://purl.obolibrary.org/obo/PATO_0000083 ## phase and create the term cropping phase of shifting cultivation, under this PATO term?

KrishnaTO replied:
Added issue for parent class 'Phase' import (#169)
Updated label and parent class to imported class. 
Commit e62bcd7e4088f6a460ab6e16ed8bb5e1ca08c75b

celineaubert commented on src/ontology/agro-edit.owl:
```diff
@@ -5446,6 +5462,127 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010155> "i
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/AGRO_00010164> "scattered trees"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/AGRO_00010164> <http://purl.obolibrary.org/obo/AGRO_00000585>)
+
+# Class: <http://purl.obolibrary.org/obo/AGRO_00010165> (taungya farming)
```
Could you please import the PATO term : http://purl.obolibrary.org/obo/PATO_0000083 ## phase and move the cropping phase of taungya term under this PATO term?

KrishnaTO replied:
Added issue for parent class 'Phase' import (#169)
Updated label and parent class to imported class. 
Commit af1d207ba51d4f8b07d0abed9ed89af9c1e10e98

celineaubert committed:
reviewed #161

reviewed the definition based on FAO definition

celineaubert committed:
reviewed #162

reviewed def and added a source

celineaubert committed:
reviewed #162

added broad syn

celineaubert committed:
move hedgerow intercropping under alley cropping

celineaubert committed:
Reviwed #166

changed the label: does not seem that ''agroforestry fallow'' term exist. There is only mentions of improved fallow in litterature.
Also, I delete the link the shifting cultivation. In shifting cultivation, the fallow phase is not managed at all (and might not include any trees) whereas in the improved fallow it is.

celineaubert commented:
Hi Krishna, I've shared my comments either by adding comments on the commit or by direclty modifying the terms in Protege. Hope this is not too messy.

KrishnaTO committed:
Merge branch 'master' into issues-147-166
